calc.py

```diff
@@ -0,0 +1,52 @@
+# This function adds two numbers
+def add(x, y):
+    return x + y
+
+# This function subtracts two numbers
+def subtract(x, y):
+    return x - y
+
+# This function multiplies two numbers
+def multiply(x, y):
+    return x * y
+
+# This function divides two numbers
+def divide(x, y):
+    return x / y
+
+
+print("Select operation.")
+print("1.Add")
+print("2.Subtract")
+print("3.Multiply")
+print("4.Divide")
+
+
+while True:
+    # take input from the user
+    choice = input("Enter choice(1/2/3/4): ")
+
+    try:
+        num1 = int(input("Enter first number: "))
+        num2 = int(input("Enter second number: "))
+    except ValueError:
+        print("Invalid input. Please enter a number.")
+        continue
+
+    if choice == '1':
+        print(num1, "+", num2, "=", add(num1, num2))
+
+    elif choice == '2':
+        print(num1, "-", num2, "=", subtract(num1, num2))
+
+    elif choice == '3':
+        print(num1, "*", num2, "=", multiply(num1, num2))
+
+    elif choice == '4':
+        print(num1, "/", num2, "=", divide(num1, num2))
+
+    # check if user wants another calculation
+    # break the while loop if answer is no
```

Comment:
```suggestion
    # exits the while loop if answer is no
```
Avoids using the same word as break whilst probably being a little more descriptive to what happens.